src/node_file.cc

```diff
@@ -885,6 +885,54 @@ static void WriteBuffer(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+// Wrapper for writev(2).
+//
+// bytesWritten = writev(fd, chunks, position, callback)
+// 0 fd        integer. file descriptor
+// 1 chunks    array of buffers to write
+// 2 position  if integer, position to write at in the file.
```

Comment:
Could, but it is a literal copy of the comment above `writeBuffer` which I didn't author. Would you still want me to fix it, and if so, in both places?